packages/ui/src/lib/Navbar/MenuItem.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, Link } from "native-base";
+import PropTypes, { InferProps } from "prop-types";
+import React from "react";
+
+export const menuItemPropTypes = {
+  name: PropTypes.string.isRequired,
+  href: PropTypes.string.isRequired,
+};
+
```

Comment:
Now we can remove the `menuItem` prefix because there is only one propTypes

Reply:
I think this one still have to be named `menuItemPropTypes` because it will be imported by `Navbar/index.tsx` like this.

```tsx
import MenuItem, { menuItemPropTypes } from "./MenuItem";
```

Reply:
if others wants to rename it, they can rename it?

Reply:
I think yes.

Reply:
I mean that doesn't seem to be the responsibility of the MenuItem

Reply:
Then let's use `propTypes` then?

Reply:
Agree